userdocs/src/introduction.md

```diff
@@ -113,26 +139,7 @@ eksctl create cluster --node-volume-size=50 --node-volume-type=io1
 
 ```
 
-!!! note
```

Comment:
This is still true for us-east-1 and many users still experience issues there.

Reply:
OK, do you think it should be in the introduction?

Reply:
Perhaps here https://eksctl.io/usage/creating-and-managing-clusters/ ?

Reply:
@martina-if done, WDYT?